neqo-qpack/src/table.rs

```diff
@@ -134,7 +134,7 @@ impl HeaderTable {
     /// `HeaderLookup` if the index does not exist in the static table.
     pub fn get_static(index: u64) -> Res<&'static StaticTableEntry> {
         let inx = usize::try_from(index).or(Err(Error::HeaderLookup))?;
-        if inx > HEADER_STATIC_TABLE.len() {
+        if inx >= HEADER_STATIC_TABLE.len() {
```

Comment:
Preference for a separate pull request, especially since this might be severe, i.e. shouldn't hide in this non-fix pull request.